accessing.md

```diff
@@ -24,7 +24,7 @@ In the example shown, we're illustrating several things:
 1. The first argument to `sql.Open` is the driver name. This is the string that the driver used to register itself with `database/sql`, and is conventionally the same as the package name to avoid confusion. For example, it's `mysql` for [github.com/go-sql-driver/mysql](https://github.com/go-sql-driver/mysql). Some drivers do not follow the convention and use the database name, e.g. `sqlite3` for [github.com/mattn/go-sqlite3](https://github.com/mattn/go-sqlite3) and `postgres` for [github.com/lib/pq](https://github.com/lib/pq).
 2. The second argument is a driver-specific syntax that tells the driver how to access the underlying datastore. In this example, we're connecting to the "hello" database inside a local MySQL server instance.
 3. You should (almost) always check and handle errors returned from all `database/sql` operations.  There are a few special cases that we'll discuss later where it doesn't make sense to do this.
-4. It is idiomatic to `defer db.Close()` if the `sql.DB` should not have a lifetime beyond the scope of the function.
+4. It is idiomatic to `defer db.Close()` , because the `sql.DB` object should not have a lifetime beyond the scope of the function.
```

Comment:
@Ely0rda 

See https://pkg.go.dev/database/sql#Open.

Based on what I've read, the `*sql.DB` value is usually long lived and will have a lifetime beyond the scope of a single function call. The proposed change explicitly indicates otherwise.

Reply:
> @Ely0rda: Sorry, I just misunderstood the phrase

No problems, the statement is heavily dependent on the reader already knowing whether the value *should* have a lifetime beyond the scope of the function. 

In the case of the main function in the example, I don't believe it should, but if you're passing a `*sql.DB` into a function then it will likely mean that you wish to have the lifetime be longer than that of a function call.

I still consider myself fairly new, so please double-check my feedback to make sure it's on point.